themes/default/build/sass/layout/_grid.scss

```diff
@@ -151,33 +127,30 @@ Styleguide 2.1.4
 	}
 }
 
-
 /*
-
-#Grid Column reverse
-
-As an alternative to push/pull when you only need to perform a simple task
-e.g if you would would like to have your sidebar on the left at desktop, but
-beneath the content at mobile, you can use `pure-g-reverse-row` to reverse the order
-
-Markup:
-<div class="pure-g show-grid pure-g-reverse-row">
-	<div class="pure-u-1-5"> Right </div>
-	<div class="pure-u-4-5"> Left </div>
-</div>
-<div class="pure-g show-grid pure-g-md-reverse-row">
-	<div class="pure-u-1-5"> Right @ md </div>
-	<div class="pure-u-4-5"> Left @ md </div>
-</div>
-
-Styleguide 2.1.5
+-------------------------------------------------------------------------
+	Grid Column reverse
+	As an alternative to push/pull when you only need to perform a simple task
+	e.g if you would would like to have your sidebar on the left at desktop, but
+	beneath the content at mobile, you can use `pure-g-reverse-row` to reverse the order
+
+	Markup:
+	<div class="pure-g show-grid pure-g-reverse-row">
+		<div class="pure-u-1-5"> Right </div>
+		<div class="pure-u-4-5"> Left </div>
+	</div>
+	<div class="pure-g show-grid pure-g-md-reverse-row">
+		<div class="pure-u-1-5"> Right @ md </div>
+		<div class="pure-u-4-5"> Left @ md </div>
+	</div>
+------------------------------------------------------------------------- 
 */
 
 @each $breakpoint, $value in $breakpoints {
 	@include respond($breakpoint) {
 		$mediaPrefix: '-' + $breakpoint + '-';
 
-		// remove the prefix for the base size
+		/* remove the prefix for the base size */
```

Comment:
This should be an inline comment so it is automatically stripped out on processing.